webapp/channels/src/components/admin_console/system_users/system_users.scss

```diff
@@ -60,8 +60,12 @@ table.systemUsersTable {
                 }
 
                 .error {
+                    max-width: 300px;
                     align-self: start;
                     grid-area: error;
+                    // this need to be here, after textElipsis because
+                    // we need to override white-space property coming from the mixing
+                    white-space: normal;
```

Comment:
Based on the comment, this should be after the include. Auto linting may have moved this?

Reply:
Not sure what caused it to move back up. I tried `npm run fix` but that doesn't do it. Anyways, fixed it now. Thanks for the sharp eyes 😛

Reply:
or we could just remove the include and write manually :P